configure.ac

```diff
@@ -153,12 +153,18 @@ AC_LINK_IFELSE(
 
 PVECLIB_DEFAULT_CFLAGS+=" $PVECLIB_FLOAT128_CFLAGS"
 
-AC_MSG_RESULT(["PVECLIB_DEFAULT base target set to '$PVECLIB_DEFAULT_CFLAGS'"])
+##### Provide a nice message is the compiler default is OK #####
+if test "$PVECLIB_DEFAULT_CFLAGS" != " "; then
```

Comment:
Should that "empty" string contain a space (`" "`) ?

Reply:
It does in this case because of the
PVECLIB_DEFAULT_CFLAGS+=" $PVECLIB_FLOAT128_CFLAGS"
includes a leading " "